kclvm/error/src/diagnostic.rs

```diff
@@ -181,6 +182,17 @@ impl fmt::Display for Level {
     }
 }
 
+/// Convert KCL diagnostic level to LSP diagnostic serverity
+impl Into<DiagnosticSeverity> for Level {
```

Comment:
I suggest moving this Into the lsp into crate to avoid error crate depending on lsp-related content.